airbyte-cdk/Dockerfile.sats

```diff
@@ -0,0 +1,5 @@
+FROM python:3.9.11-alpine3.15
```

Comment:
do we know if/why SATs can't run on 3.9-slim? I agree that using a single image for "almost" everything would be better

Reply:
Just hit a small snag with a dependency (`apk`) so decided to punt on consolidating them until we were pretty sure we wanted to. I'll go ahead and see what it looks like to use a single base image.

Reply:
Resolving this as with the latest update we're not modifying existing Dockerfiles.